lib/postgres_to_redshift/key.rb

```diff
@@ -40,6 +40,14 @@ def target_column_name
       decomposed_key[:target_column_name]
     end
 
+    def touches_table?(name)
+      [target_table_name, table_name].compact.map(&:downcase).include?(name.downcase)
+    end
```

Comment:
I am trying to follow this 🤔  give me one second to try to come up with my own answer or a question 😂

Reply:
Ah, nevermind, I got my own answer, I was reading this wrong 👍 (classic gus)
Looks good to me, going to approve 🙌

Reply:
Not sure why, for a few minutes I was reading this like "does any of the tables include the key description?" 😅